packages/types/src/market.ts

```diff
@@ -40,3 +40,35 @@ export type FindAllMarketArgs = {
 export type MarketCapResult = {
   [k: string]: MarketData
 }
+
+export enum SupportedFiatCurrencies {
```

Comment:
Based on the requirements, `SupportedFiatCurrencies` should be in string union form ie, `'USD' | 'EUR'`